package.json

```diff
@@ -54,6 +46,7 @@
     "@types/wicg-file-system-access": "^2023.10.5",
     "jest": "^29.7.0",
     "jsdom-testing-mocks": "^1.13.0",
+    "memfs": "^4.11.0",
```

Comment:
What's this for?

Reply:
https://github.com/gravitational/teleport/blob/master/docs/check-redirects/check-redirects.test.js#L1

Reply:
Yeah, I'm working on moving it to a separate `package.json`. Unfortunately, pnpm doesn't treat this package.json as an isolated entity (although it is not added in `pnpm-workspace.yaml`) so when I run `pnpm install` from `docs/check-redirect` it installs all webapps dependencies.
I will look tomorrow for a way to overcome this (I believe incorrect) behavior.

Reply:
This makes sense to me! Let me know if you'd like me to edit `docs/check-redirect` at all. While I'd like to add a workflow to run this during CI at some point to prevent 404s from the Web UI etc., there's nothing that currently runs the redirect checker script, and it must be run manually. I can also move it to another repo if that makes sense.